### ISSUE TITLE
imagesLoaded not being called

### ISSUE THREAD

**Issue by winkler1** [eiriklv/react-masonry-mixin#16]
Hey Eirik!

Am upgrading code after way too long. Running into some glitches with a cold cache. In versions 0.5.7 and up, imagesloaded does not fire-- this never logs:

```
imagesLoaded: function() {
    imagesloaded(this.refs[reference].getDOMNode(), function(instance) {
        console.log(">> Images were Loaded!");
        this.masonry.layout();
    }.bind(this));
},
```

0.5.5 looks fine, so I'll probably stick with that...

**Comment by eiriklv** [comment]
Thanks for the heads up @winkler1. Will look into it :-)

**Comment by eiriklv** [comment]
@winkler1 could you try 1.0.2? And which version of React are you using?

**Comment by winkler1** [comment]
Issue is still present. Using "react": "^0.13.0". It's not a problem with getDOMNode, the node was being set... traced through the imageLoader code a bit and found that it was finding the child <img>s, but for some reason the callback is never invoked.

**Comment by eiriklv** [comment]
That's strange.. I can

**Comment by eiriklv** [comment]
If you find any more clues be sure to post it :-)

**Comment by FelipeMarcos** [comment]
I was having this issue as well. I fixed it by adding `this.imagesLoaded();` to `componentDidMount` because `componentDidUpdate` is not called for the initial render.

**Comment by eiriklv** [comment]
Thanks @FelipeMarcos! Will release a fix.

**Comment by eiriklv** [comment]
released as 1.0.3
